deck-builder/index.html

```diff
@@ -0,0 +1,20 @@
+<!Doctype html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <title>Weiss Schwarz Deck Builder</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Open+Sans" />
+  </head>
+  <body>
+    <header>
+      <a href=""><img src="weiss-schwarz-logo.png"
+        alt="Weiss Schwarz Logo" /></a>
+    </header>
+    <h1>Please Select Your Packs</h1>
+    <div id="booster-packs-section">
```

Comment:
It would be more semantic to use a `section` element with an `id` of `booster-packs`.